crates/ide/src/status.rs

```diff
@@ -51,19 +60,19 @@ pub(crate) fn status(db: &RootDatabase, file_id: Option<FileId>) -> String {
                     Some(it) => format!("{}({:?})", it, krate),
                     None => format!("{:?}", krate),
                 };
-                format_to!(buf, "crate: {}\n", display_crate(krate));
+                format_to!(buf, "Crate: {}\n", display_crate(krate));
                 let deps = crate_graph[krate]
                     .dependencies
                     .iter()
                     .map(|dep| format!("{}={:?}", dep.name, dep.crate_id))
                     .format(", ");
-                format_to!(buf, "deps: {}\n", deps);
+                format_to!(buf, "Dependencies: {}\n", deps);
             }
-            None => format_to!(buf, "does not belong to any crate"),
+            None => format_to!(buf, "Does not belong to any crate"),
         }
     }
 
-    buf
+    buf.trim().to_string()
```

Comment:
Is the trim needed? A trailing newline is probably fine.

Reply:
Purely for consistency. It needs to be there as it removes the double trailing newlines.

Reply:
FWIW, we can truncate it in-place with `buf.truncate(s.trim_end().len());`

Reply:
From the looks of that that only removes trailing double newlines at the end of the string?

Reply:
Yeah, do we have newlines at the beginning?

Reply:
It is possible for that to happen.

Reply:
Okay, never mind then.